cabal-testsuite/cabal-testsuite.cabal

```diff
@@ -67,16 +67,16 @@ library
     , directory             ^>= 1.2.0.1 || ^>= 1.3.0.0
     , exceptions            ^>= 0.10.0
     , filepath              ^>= 1.3.0.1 || ^>= 1.4.0.0
-    , network-wait          ^>= 0.1.2.0
-    , optparse-applicative  ^>= 0.14.3.0 || ^>=0.15.1.0 || ^>=0.16.0.0
+    , network-wait          ^>= 0.1.2.0 || ^>= 0.2.0.0
+    , optparse-applicative  ^>= 0.14.3.0 || ^>=0.15.1.0 || ^>=0.16.0.0 || ^>= 0.17.0.0
     , process               ^>= 1.2.1.0 || ^>= 1.4.2.0 || ^>= 1.6.1.0
     , regex-base            ^>= 0.94.0.1
     , regex-tdfa            ^>= 1.2.3.1 || ^>=1.3.1.0
     , retry                 ^>= 0.9.1.0
     , array                 ^>= 0.4.0.1 || ^>= 0.5.0.0
     , temporary             ^>= 1.3
-    , text                  ^>= 1.2.3.1
     , transformers          ^>= 0.3.0.0 || ^>= 0.4.2.0 || ^>= 0.5.2.0
+    , text                  ^>= 1.2.3.1 || ^>= 2.0.1
```

Comment:
Maybe better to keep the original lexical ordering?

Reply:
You are right.  Something went wrong with the hunk selection in emacs magit, my subconscience noticed it, but it didn't verbalize in my conscience, and so I went ahead... ;-)